src/main/java/seedu/duke/parsermanager/delete/CommandDeletePropertyParser.java

```diff
@@ -11,13 +11,18 @@
 import seedu.duke.exception.parsedeleteexception.parsedeletepropertyexception.InvalidDeletePropertyIndexException;
 import seedu.duke.exception.parsedeleteexception.parsedeletepropertyexception.MissingDeletePropertyFlagException;
 import seedu.duke.exception.parsedeleteexception.parsedeletepropertyexception.ParseDeletePropertyException;
+import seedu.duke.parsermanager.Parser;
 
 import java.util.ArrayList;
 
 import static seedu.duke.CommandStructure.CHECK_PROPERTY_FLAGS;
+import static seedu.duke.CommandStructure.START_INDEX;
 import static seedu.duke.Messages.MESSAGE_NOT_INTEGER;
 
-public class CommandDeletePropertyParser extends Parser {
+/**
+ * Parser for delete property command.
+ */
+public class CommandDeletePropertyParser extends CommandDeleteParser {

```

Comment:
Good job adding the documentations. Would be good if you fix the format since it's required to start with words such as "Parses" or "Fixes" etc